### DISCUSSION TITLE
Accessibility / Color blindness - Online / Offline status

### DISCUSSION THREAD

**Discussion by acseven** [rustdesk/rustdesk#9716] [General]
If possible, I would like to request modification of the online/offline status circle for those with deuteranopia; the status circle is very small and many times indistinguishable for those who have this condition like me. 

![image](https://github.com/user-attachments/assets/6f0192ad-0c6e-49f1-97dd-4e18083b716f)

I would suggest (as an option for example), an "X" / checkmark icon instead or at least make the circle quite a bit larger which can help out a bit. 

Thank you for this project!

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/9544
